frontend/pages/index.tsx

```diff
@@ -13,14 +14,132 @@ import {
   imageSx,
   primaryTranslucentOverlaySx,
   secondaryTranslucentOverlaySx,
+  spacing2,
+  marTop5,
+  marTop10,
+  marLeft10,
+  marRight10,
+  largeFont,
+  impactText,
+  gradientSection,
+  buttonSecondary,
 } from '../styles/indexStyles';
+import Image from 'next/image';
 
 const Home: NextPage = () => {
   return (
     <div>
       <Head>
-        <title>Home</title>
+        <title>Giving Coupons</title>
       </Head>
+      <Box sx={hero}>
```

Comment:
Let's add a line break between same level tags for consistency.
```suggestion
      </Head>
      
      <Box sx={hero}>
```